docs/content/features/exploiters/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Exploiters"
+chapter: true
+pre: "<i class='fas fa-ethernet'></i> "
+---
+
+# Exploiters
+
+##
+
+In a network breach, the exploitation phase is a critical stage where attackers
```

Comment:
IMHO this is an unnecessary intensifier. See https://www.wordrake.com/blog/delete-intensifiers-and-qualifiers or https://www.elegantlegalwriting.com/p/delete-unnecessary-modifiers-especially for more opinions on the subject.

```suggestion
In a network breach, the exploitation phase is when attackers
```